src/jobs/deploy-s3.yml

```diff
@@ -1,9 +1,80 @@
-description: >
+description: |
   This job deploys an application to aws s3.
 
 executor: node/default
 
+parameters:
+  source:
+    type: string
+    default: SOURCE
+    description: |
+      a local directory path to sync with s3
+
+  s3_bucket:
+    type: string
+    description: |
+      A URI to an S3 bucket
+
+  tag:
+    type: string
+    description: |
+      tag for the repo
+
+  aws_region:
+    type: env_var_name
+    default: AWS_REGION_N_VA
+    description: |
+      Specify the region for aws
+
+  aws_access_key_id:
+    default: AWS_ACCESS_KEY_ID
+    type: env_var_name
+    description: |
+      AWS access key id for IAM role. Set this to the name of
+      the environment variable you will use to hold this
+      value, i.e. AWS_ACCESS_KEY.
+
+  aws_secret_access_key:
+    default: AWS_SECRET_ACCESS_KEY
+    type: env_var_name
+    description: |
+      Secret Key for MACHINE_AWS_ECS
+
+  mentions:
+    type: env_var_name
+    default: MENTIONS
+    description: |
+      MENTIONS is a the environment variable name for the slack context.
+
 steps:
+  - checkout
+  - tag-repo:
```

Comment:
I would communicate with @JosueMolinaMorales to determine how merging should work/what other fields you may or may not need